pyanaconda/payload/__init__.py

```diff
@@ -863,29 +863,29 @@ def _setup_install_device(self, checkmount):
         metalink = None
 
         # See if we already have stuff mounted due to dracut
-        isodev = payload_utils.get_mount_device(DRACUT_ISODIR)
-        device = payload_utils.get_mount_device(DRACUT_REPODIR)
+        iso_device_path = payload_utils.get_mount_device(DRACUT_ISODIR)
+        device_path = payload_utils.get_mount_device(DRACUT_REPODIR)
```

Comment:
Rename this to `repo_device_path` instead.

Reply:
Fixed.